bin/tat

```diff
@@ -2,7 +2,8 @@
 #
 # Attach or create tmux session named the same as current directory.
 
-session_name="$(basename "$PWD" | tr . -)"
+path_name="$(basename "$PWD" | tr . -)"
+session_name=${1-$path_name}
```

Comment:
In conclusion: it seems like my suggestion doesn't change the behavior. I'd still prefer `:-` to `-`, just because `:-` is more common, I think.